ovs/lib/albanodecontroller.py

```diff
@@ -210,3 +225,78 @@ def model_local_albanode(**kwargs):
                 node.password = main_config['password']
                 node.storagerouter = StorageRouterList.get_by_ip(main_config['ip'])
                 node.save()
+
+    @staticmethod
+    def checkup_maintenance_agents():
+        """
+        Check if requested nr of maintenance agents / backend is actually present
+        Add / remove as necessary
+        :return: None
+        """
+        service_template_key = 'ovs-alba-maintenance_{0}-{1}'
+        maintenance_agents_map = {}
+        asd_nodes = AlbaNodeList.get_albanodes()
+        nr_of_storage_nodes = len(asd_nodes)
+
+        def get_node_load(backend_name):
+            highest_load = 0
+            lowest_load = 100
```

Comment:
You could just set this to ```None```, and below use ```if actual_nr_of_agents < lowest_load or lowest_load is None:```. Then there never will be an issue where 100 would be insufficient. Or use ```sys.maxint```.